jwala-common/src/main/java/com/cerner/jwala/common/properties/PropertyKeys.java

```diff
@@ -32,7 +32,8 @@
     PORT("ssh.port"),
     PRIVATE_KEY_FILE("ssh.privateKeyFile"),
     KNOWN_HOSTS_FILE("ssh.knownHostsFile"),
-    ENCRYPTED_PASSWORD("ssh.encrypted.password");
+    ENCRYPTED_PASSWORD("ssh.encrypted.password"),
+    MOD_PROXY_OVERRIDE("mod_proxy_override");
```

Comment:
this is a bad name for the property... should be renamed to something like "jvm.route.max.length"